dev/tests/integration/phpunit.xml.dist

```diff
@@ -19,7 +19,6 @@
         <testsuite name="Magento Integration Tests">
             <directory suffix="Test.php">testsuite</directory>
             <directory suffix="Test.php">../../../update/dev/tests/integration/testsuite</directory>
-            <exclude>testsuite/Magento/Test/Integrity</exclude>
```

Comment:
Makes sense.
Though, I'd wish to have results of integrity tests to be separate, it's not a big difference and may be a subjective opinion.
I din't realize these tests take just 1min. So, for sure, your idea of combining them with other integration test makes sense.